huaweicloud/services/vpc/resource_huaweicloud_vpc.go

```diff
@@ -141,15 +142,24 @@ func resourceVirtualPrivateCloudV1Create(d *schema.ResourceData, meta interface{
 	return resourceVirtualPrivateCloudV1Read(d, meta)
 }
 
-func resourceVirtualPrivateCloudV1Read(d *schema.ResourceData, meta interface{}) error {
-	config := meta.(*config.Config)
-	vpcClient, err := config.NetworkingV1Client(GetRegion(d, config))
+// GetVpcFromServer is a method to obtain vpc informations through vpc ID.
+func GetVpcFromServer(d *schema.ResourceData, config *config.Config, vpcId string) (*vpcs.Vpc, error) {
```

Comment:
suggest to define the method as `GetVpcByID`

Reply:
The function name has been updated.